geopandas/io/sql.py

```diff
@@ -67,4 +69,64 @@ def load_geom(x):
     return GeoDataFrame(df, crs=crs, geometry=geom_col)
 
 
+def get_srid(crs):
+    """
+    Returns the SRID from a CRS dict or string, if available.
+
+    Parameters
+    ----------
+    crs : dict or str
+
+    Returns
+    -------
+    srid : str
+        If unsuccessful, returns None.
+    """
+
+    if not crs:
+        return
+
+    if isinstance(crs, str):
+        crs = fiona.crs.from_string(crs)
+
+    if 'init' in crs.keys():
+        if 'epsg' in crs['init']:
+            return crs['init'].split('epsg:')[1]
+
+
 read_postgis = functools.partial(read_sql, hex_encoded=True)
+
+
+def to_postgis(df, name, con, hex_encoded=True, **kwargs):
+    """
+    Writes a geodataframe to a postgis database.
+
+    Parameters
+    ----------
+    df : geodataframe
+    name : str
+        Name of table in database
+    engine : SQLAlchemy engine
+
+    **kwargs are passed to pandas.to_sql; see documentation for available
+    parameters
+    """
+
+    geom = df.geometry
+    temp_df = df.copy()
+
+    def convert_geometry(geom):
+        return shapely.wkb.dumps(geom, hex=hex_encoded)
```

Comment:
made the comment in the wrong area; is there ar reason we're using the dumps function rather than `wkb_hex` attribute?

Reply:
I _think_ that setting `hex=False` here might be desirable for interacting with some types of spatial databases. However, I've only really worked with postgis, so I'm not confident in that. Certainly with the geometry and SRID setting command as written, other changes would be necessary to support writing to a non-postgis database right now, so maybe this attempt at laying groundwork for future development is more trouble than it is worth.

Reply:
Ah, right, how postgis-centric of me. I'd keep it like this, since just having the option may be useful in future.

Reply:
Though, if this is named `postgis` should it be generalized to `to_sql`? I think this might work in `spatialite`?